test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -259,6 +259,12 @@ export default class BaseFormObject {
     return errors[index];
   }
 
+  getErrorBoxTitle() {
```

Comment:
I would either rename this to something more generic (i.e. do not include ErrorBox since the result of the function would not matter if its in an alert box) or update the function to first find by role using "alert" and then within the alert box search for the Title/heading element.

Reply:
oh yeah good call!